pkg/machines/libvirt-common.js

```diff
@@ -796,6 +796,9 @@ export function parseStoragePoolDumpxml(connectionName, storagePoolXml, id_overw
     result['type'] = storagePoolElem.getAttribute('type');
     result['name'] = storagePoolElem.getElementsByTagName('name')[0].childNodes[0].nodeValue;
     result['id'] = id_overwrite || storagePoolElem.getElementsByTagName('uuid')[0].childNodes[0].nodeValue;
+    result['capacity'] = storagePoolElem.getElementsByTagName('capacity')[0].childNodes[0].nodeValue;
+    result['available'] = storagePoolElem.getElementsByTagName('available')[0].childNodes[0].nodeValue;
```

Comment:
I don't think that the XML is updated on the fly when the available space changes. This a a volatile information that should be fetched with an API